manifests/v/VisIt/VisIt/3.3.3/VisIt.VisIt.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: VisIt.VisIt
+PackageVersion: 3.3.3
+PackageLocale: en-US
+Publisher: VisIt

```

```suggestion
Publisher: LLNL
```
This should match control panel